tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -235,6 +235,29 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, GetColumnUsingTabs )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, GetColumnUsingTabs" );
+
+	std::string inString1 = " Col1 \t Col2 \t Col3 ";
+	EXPECT_EQ( " Col1 ", GetColumnUsingTabs( inString1, 1 ) );
+	EXPECT_EQ( " Col2 ", GetColumnUsingTabs( inString1, 2 ) );
+	EXPECT_EQ( " Col3 ", GetColumnUsingTabs( inString1, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString1, 4 ) );
+
+	std::string inString2 = "Col1\tCol2\tCol3";
+	EXPECT_EQ( "Col1", GetColumnUsingTabs( inString2, 1 ) );
+	EXPECT_EQ( "Col2", GetColumnUsingTabs( inString2, 2 ) );
+	EXPECT_EQ( "Col3", GetColumnUsingTabs( inString2, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString2, 4 ) );
+
+	std::string inString3 = "Col1\tCol2\tCol3\t";
+	EXPECT_EQ( "Col1", GetColumnUsingTabs( inString3, 1 ) );
+	EXPECT_EQ( "Col2", GetColumnUsingTabs( inString3, 2 ) );
+	EXPECT_EQ( "Col3", GetColumnUsingTabs( inString3, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString3, 4 )  );
```

Comment:
@Myoldmopar I added those new test cases and I think I captured the correct behavior of this function.